edsl.md

```diff
@@ -37,11 +37,13 @@ where `F1 : F2 : F3 : F4` is the (two's complement) byte-array representation of
     syntax TypedArg ::= #uint160 ( Int )
                       | #address ( Int )
                       | #uint256 ( Int )
+                      | #uint8   ( Int )
                       | #int256  ( Int )
                       | #int128  ( Int )
                       | #bytes32 ( Int )
                       | #bool    ( Int )
                       | #bytes   ( Int , Int )
+                      | #array   ( TypedArg , Int , TypedArgs )
  // ------------------------------------------
```

Comment:
@tomtomjhj can you please extend the `-----` underneath the definition to extend to the same length as the line above? For the reason please see the "Broken Window Effect" https://en.wikipedia.org/wiki/Broken_windows_theory.

The idea is that if someone sees some small part of the code not conforming to the style, they may be more inclined to write more code which doesn't conform to the style.